### PR TITLE
Mrc 477 Provide unfiltered indicator data to map component

### PR DESCRIPTION
**Pull request by EmmaLRussell**
Contains changes from https://github.com/mrc-ide/hint/pull/46

### CODE REVIEW

File: src/app/static/src/tests/filteredData/getters.test.ts [hunk 1]
```diff
@@ -126,4 +124,258 @@ describe("FilteredData mutations", () => {
...
+                area_id: "area1",
+                ancrt_test_pos: 2
+            },
+            {
+                iso3: "MWI",
+                area_id: "area2",
+                ancrt_test_pos: 3
+            }
+        ];
+        const testRootState = {
```
Comment (hillalex) [inline src/app/static/src/tests/filteredData/getters.test.ts]: This testState object is mostly the same each time, I've added a helper in another branch that can tidy this up a bit, might be nice to bring it in here:
https://github.com/mrc-ide/hint/pull/56/files#diff-06da03af1436a7e713350bccd618bf8bR61

File: src/app/static/src/app/components/plots/Choropleth.vue [hunk 2]
```diff
@@ -6,7 +6,9 @@
             </l-geo-json>
         </template>
         <map-control @indicator-changed="onIndicatorChange"
-                     @detail-changed="onDetailChange"></map-control>
+                     @detail-changed="onDetailChange"
+                     :indicator="indicator"
```
[the remark this answers is not in the document]
Comment (EmmaLRussell) [reply src/app/static/src/app/components/plots/Choropleth.vue]: Ah well spotted!

File: src/app/static/src/app/store/filteredData/getters.ts [hunk 3]
```diff
@@ -35,5 +36,89 @@ export const getters = {
...
+        const data =  getUnfilteredData(state, rootState);
+        if (!data) {
+            return null;
+        }
+
+        const result = {} as {[k: string]: Indicators};
+        const artRange = {min: null, max: null} as IndicatorRange;
+        const prevRange = {min: null, max: null} as IndicatorRange;
+
+        const updateRange = (r: IndicatorRange, value: number) => {
```
[the remark this answers is not in the document]
Comment (EmmaLRussell) [reply src/app/static/src/app/store/filteredData/getters.ts]: OK, following Friday's discussion, I've added a ticket to take this out and replace it with getting the range from the API: https://vimc.myjetbrains.com/youtrack/issue/mrc-504
Leaving as is for the moment.

File: src/app/static/src/app/store/filteredData/getters.ts [hunk 4]
```diff
@@ -19,24 +19,9 @@ export const getters = {
...
-                    ]},
-                {name: "Southern Region", id: "MWI.3", options: [
-                        {name: "Balaka", id: "MWI.3.1"}
-                    ]}
-            ];
-        }
-        return result;
+        return shape && shape.filters &&
+                        shape.filters.regions &&
+                        (shape.filters.regions as any).options ? (shape.filters.regions as any).options : [];
```
Comment (EmmaLRussell) [inline src/app/static/src/app/store/filteredData/getters.ts]: These filters have an additional level 'options' now to get to the top level, however I haven't regenerated types from the latest hintr yet as assume will have same problem with recursion as last time...

**Pull request merged**